modules/machineset-yaml-gcp.adoc

```diff
@@ -58,15 +89,25 @@ spec:
             creationTimestamp: null
           networkInterfaces:
           - network: <infrastructureID>-network <1>
-            subnetwork: <infrastructureID>-<role>-subnet <2>
+            subnetwork: <infrastructureID>-<role>-subnet <1> <2>
+ifndef::infra[]
           projectID: <project_name> <4>
+endif::infra[]
+ifdef::infra[]
+          projectID: <project_name> <5>
+endif::infra[]
           region: us-central1
           serviceAccounts:
+ifndef::infra[]
           - email: <infrastructureID>-w@<project_name>.iam.gserviceaccount.com <1> <4>
+endif::infra[]
+ifdef::infra[]
+          - email: <infrastructureID>-w@<project_name>.iam.gserviceaccount.com <1> <5>
+endif::infra[]
             scopes:
             - https://www.googleapis.com/auth/cloud-platform
           tags:
-          - <infrastructureID>-<role> <2>
+          - <infrastructureID>-<role> <1> <2>
```

Comment:
should be <infrastructureID>-woker, If it is infra, the machine will fail to create

Reply:
@sunzhaohua2 Is this true for machine sets on Infra and GCP?

Reply:
@mburke5678 yes, I think so.